services/common/src/redux/selectors/projectSelectors.ts

```diff
@@ -16,10 +17,18 @@ export const {
   getProjectLinks,
 } = projectReducer;
 
+const formatProjectSummaryAgent = (agent): IParty => {
+  if (!agent || !agent.party_guid) {
+    return agent;
+  }
+  return { ...agent, address: agent.address[0] };
+};
+
 export const getFormattedProjectSummary = createSelector(
   [getProjectSummary, getProject],
   (summary, project) => {
-    let formattedSummary = { ...summary, authorizationOptions: [] };
+    const agent = formatProjectSummaryAgent(summary.agent);
```

Comment:
Will this throw an error if agent is null or undefined? I can't remember if strict null checks is enabled in our code base or not.

Reply:
agent doesn't have to be defined, but summary does. I think it must always be defined because it throws no errors on new entries.